source/definition_of_done.rst

```diff
@@ -0,0 +1,10 @@
+Definition of Done
+"""""""""""""""""""
+
+The following requirements must be satisfied before a task can be considered done:
+
+* The solution is in agreement with the design
+* 100% test coverage
```

Comment:
I do not have any experience with the requirement of 100% test coverage. To me its seems something that might cost more than we gain?  We could loosen up the language and say "good enough" testing, ca. 70-90%? 

I will of course accept it, but I guess I am just saying I have no idea how much work that will be to achieve.

Reply:
I think 100% coverage of the unit tests is quite important. It requires some extra effort by everyone, but my experience is that it is extremely useful in the long run... This is basically also what we already agreed about in the definition-of-done.

Reply:
Yes, thats true, I was just thinking this through once more. Anyway, seems like I am in minority on this, so I will just accept it!